styles/globals.css

```diff
@@ -0,0 +1,332 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;0,400;0,500;0,700;1,300;1,400;1,500;1,700&display=swap");
+
+:root {
+  --primary-green: #0d712e;
+  --primary-yellow: #ffd900;
+  --primary-blue: #022777;
+  --primary-white: #f5f5fa;
+
+  --secundary-blue: #1c3183;
+  --secundary-dark-green: #00450e;
+  --secundary-green: #0b6128;
+  --secundary-light-green: #069736;
+  --secundary-yellow: #1c3183;
+  --secundary-red-warning: #ed3233;
+
+  --tertiary-light-blue: #b0c4d5;
+  --tertiary-dark-blue: #104b7e;
+  --tertiary-yellow: #ffcb21;
+  --tertiary-background: rgba(194, 18, 18, 0.15);
+
+  --neutral-black: #1a1a1a;
+  --neutral-dark: #525252;
+  --neutral-medium-dark: #525252;
+  --neutral-medium: #9b9b9f;
+  --neutral-medium-light: #c7c8d0;
+  --neutral-light: #f2f2f7;
+  --neutral-white: #ffffff;
+}
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 32px;
+}
+
+html,
+body {
+  max-width: 100vw;
+  overflow-x: hidden;
+}
+
+.display,
+h1,
+h2,
+h3,
+h4,
+h5 {
+  font-weight: 700;
+}
+
+.display {
+  font-size: 58px;
+}
+
+h1 {
+  font-size: 64px;
+}
+
+h2 {
+  font-size: 56px;
+}
+
+h3 {
+  font-size: 46px;
+}
+
+h4 {
+  font-size: 38px;
+}
+
+h5 {
+  font-size: 28px;
+}
+
+h5.uppercase {
+  text-transform: uppercase;
+}
+
+.b1 {
+  font-size: 32px;
+}
+
+.b2 {
+  font-size: 24px;
+}
+
+.b3 {
+  font-size: 24px;
+  font-weight: 500;
+}
+
+.b4 {
+  font-size: 24px;
+}
+
+.b5 {
+  font-size: 24px;
+  font-weight: 300;
+}
+
+button,
+a {
+  font-size: 24px;
+  font-weight: 700;
+}
+
+button {
+  color: var(--neutral-white);
+  padding: 8px 33.5px;
+  border-radius: 40px;
+  border: 1px solid;
+  line-height: 24px;
+}
+
+button.primary {
+  background-color: var(--primary-blue);
+}
+
+button.primary:hover {
+  background-color: #374399;
+}
+
+button.primary:active {
+  background-color: #002167;
+}
+
+button.primary.outline {
```

Comment:
If you are creating styles for the buttons, you should create a button component and the css module for it as well

Reply:
It is following the style guide. I thought it will be used for all buttons.

Reply:
I don't know that yet. It is a general styling based on the style guide
![Screenshot 2023-02-27 at 10 25 19 PM](https://user-images.githubusercontent.com/20172847/221745658-486de26f-ca15-4753-8a73-449f64f93319.png)

Reply:
Gotcha. let's keep it here and move it to a module file when we create the button component. 
Great job :)